docs/mochitests.md

```diff
@@ -241,13 +241,13 @@ If symbolic link is suddenly lost between debugger.html and Firefox source, in y
 2. Execute `./mach mochitest --headless devtools/client/debugger/new`
 
   If a symbolic link occurs, error message(s) will be displayed.
-  
+
   ![Test harness with symbolic link errors](http://i40.photobucket.com/albums/e250/md2k6/Public/Opensource/debugger-html-6297/mochitest-error_zpsgicbau0z.jpg)
 
 3. Execute `./bin/prepare-mochitests-dev`.
 
 Running this command again allows the preparation script to check the integrity of the firefox directory and all symbolic
-links. It will then automatically execute `yarn copy-assets-watch` on your behalf, which ensures the symbolic linking process
+links. It will then automatically execute `yarn copy` on your behalf, which ensures the symbolic linking process
```

Comment:
I'm fairly certain this doesn't start a `watch` anymore, I think it just copies the assets over once.